test/operators/binary_arithmetic.test.cpp

```diff
@@ -278,6 +278,10 @@ TEST_CASE("Multiplication of big numbers",
     num1 = big_pow10(19876);
     num2 = big_pow10(23450);
     REQUIRE(num1 * num2 == big_pow10(43326));
+
+    num1 = big_pow10(19876) + 1;
+    num2 = big_pow10(23450);
+    REQUIRE(num1 * num2 == big_pow10(43326) + big_pow10(23450));
```

Comment:
This test seems confusing at first glance. We can have tests like this when `+` or `-` operators are optimised. For now, you may remove it.

Reply:
So should I leave it without a test for this case?

Reply:
You can remove these 4 lines. The code coverage won't drop as tests with powers of 10 already exist.

Reply:
Coverage does drop without the last commit.
There are tests multiplying powers of 10 but then it uses only the LHS(with _*this_ ). So the RHS(with _num_ ) is unused and coverage drops.

Reply:
You're right. Let it be then.